Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi9:9.1.0-1750
+
+RUN dnf install -y dnf-plugins-core-4.1.0-3.el9.noarch make-1:4.3-7.el9.x86_64 && \
```

Comment:
are we going to install other requirements in the subsequent PRs?

Reply:
yup.  You can submit a PR if you feel comfortable with the code base.